portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,6 +61,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             long timestamp = (long) commentEntity.getProperty("timestamp");
             Comments newComment =  new Comments(id, comment, name, timestamp);
             comments.add(newComment);
+            count_comments++;
+            if (count_comments >= no_of_comments ) {
```

Comment:
You can actually have "if (++count_comments >= comment_count) {" here and remove line 64.

Reply:
I have resolved all the other changes required.